pkg/virt-api/rest/subresource.go

```diff
@@ -472,6 +472,13 @@ func (app *SubresourceAPIApp) StartVMRequestHandler(request *restful.Request, re
 		writeError(errors.NewConflict(v1.Resource("virtualmachine"), name, fmt.Errorf("VM is already running")), response)
 		return
 	}
+	if vm.Status.VolumeUpdateState != nil &&
+		vm.Status.VolumeUpdateState.VolumeMigrationState != nil &&
+		vm.Status.VolumeUpdateState.VolumeMigrationState.ManualRecoveryRequired != nil &&
+		*vm.Status.VolumeUpdateState.VolumeMigrationState.ManualRecoveryRequired {
+		writeError(errors.NewConflict(v1.Resource("virtualmachine"), name, fmt.Errorf("VM needs to be manually recovered because the volume migration failed and the volumes are in an incosistent state")), response)
```

Comment:
maybe something like "VM recovery required: Volume migration failed, leaving some volumes pointing to non-existent targets; manual intervention is needed to reassign them to their original volumes." ?